code/modules/karma/karma.dm

```diff
@@ -198,6 +198,7 @@ You've gained <b>[totalkarma]</b> total karma in your time here.<br>"}
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=1'>Unlock Machine People -- 15KP</a><br>
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=2'>Unlock Kidan -- 30KP</a><br>
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=3'>Unlock Grey -- 30KP</a><br>
+			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=3'>Unlock Drask -- 30KP</a><br>
 			<a href='?src=\ref[src];karmashop=shop;KarmaBuy2=4'>Unlock Vox -- 45KP</a><br>
```

Comment:
Why's this got the same KarmaBuy2=<value> as Greys? Won't that cause issues-- like if you get Grey, you get Drask as well, and vise-versa?